README.md

```diff
@@ -1,25 +1,25 @@
 # Microsoft Workers for Climate Justice
 
-Yesterday, a major collaboration between Microsoft, Chevron and Schlumberger, an oilfield services company, was [announced](https://news.microsoft.com/2019/09/17/schlumberger-chevron-and-microsoft-announce-collaboration-to-accelerate-digital-transformation/). This partnership outlines Microsoft's use of ["a cloud-based artificial intelligence platform to improve digital services in the oil field.”](https://www.chron.com/business/energy/article/Schlumberger-Chevron-and-Microsoft-launch-14445539.php) With this kind of news making headlines just 3 days before the Global Climate Strike, it's no longer possible for us to ignore Microsoft's complicity in the climate crisis. 
+Yesterday, a major collaboration between Microsoft, Chevron and Schlumberger, an oilfield services company, was [announced](https://news.microsoft.com/2019/09/17/schlumberger-chevron-and-microsoft-announce-collaboration-to-accelerate-digital-transformation/). This partnership outlines Microsoft's use of ["a cloud-based artificial intelligence platform to improve digital services in the oil field.”](https://www.chron.com/business/energy/article/Schlumberger-Chevron-and-Microsoft-launch-14445539.php) This news just days before the Global Climate Strike is a timely example of the awful clash between our economic system and survival of the planet.
 
 [The Intergovernmental Panel on Climate Change](https://report.ipcc.ch/sr15/pdf/sr15_spm_final.pdf) (IPCC) declares 2030 as the tipping point for irreversible damage to our planet. It is imperative for industry leaders such as Microsoft to respond to this emergency as soon as possible and set an example for other tech companies. This is humanity's biggest challenge, requiring the contributions from corporations, organizations, and government to enact systemic change.
 
 The youth have realized that climate change is a threat insurmountable by individual action and have [called](https://www.theguardian.com/commentisfree/2019/may/23/greta-thunberg-young-people-climate-strikes-20-september) upon the rest of the world to support them. We must use the Global Climate Strike, inspired by 16-year-old Swedish climate activist [Greta Thunberg's school strikes](https://www.fridaysforfuture.org/), as an opportunity to stand together and demand the transformation that our world needs. Doing so is not only taking responsibility for the mistakes that future generations will have to pay for but also the only viable way forward.
 
-But first, we have to acknowledge our company's role in our warming planet. 
+Our company should reconsider its technology support of oil and gas exploitation, and should have a deliberate plan to withdraw our innovation from that sector, as a matter of principal.
```

Comment:
I agree with this statement, but I think it occurs too early in this document. The original statement starts the process of building a case for this argument. A statement such as this should come later, after the case has been built